modules/virt-vm-storage-volume-types.adoc

```diff
@@ -30,6 +30,10 @@ PVC.
 DataVolumes build on the *persistentVolumeClaim* disk type by managing the process
 of preparing the virtual machine disk via an import, clone, or upload operation.
 VMs that use this volume type are guaranteed not to start until the volume is ready.
++
+Define DataVolumes with a `type` of *dataVolume*. If another volume type is
+used in error, such as *PersistentVolumeClaim*, a warning will be triggered
+and the virtual machine will fail to start.
```

Comment:
I'm confused why this assembly doesn't contain a sample yaml VM definition. IDK how a user's supposed to use pre-generated YAML without an example.

As is, I'm not sure that this module belongs in this assembly because "storage volume type" isn't mentioned anywhere else, so IDK when a user would need to know this information.

It seems like this information needs to go wherever it is that a user crafts the YAML definition, not here. Maybe https://cnv-4562_2--ocpdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/importing_vms/virt-importing-virtual-machine-images-datavolumes.html?

Reply:
I agree with @kalexand-rh . I tried the following YAML files, I didn't see the failure. VM started.

DataVolume specification
```
$ cat import-dv-cirros.yaml
apiVersion: cdi.kubevirt.io/v1alpha1
kind: DataVolume
metadata:
  name: import-dv-cirros
spec:
  source:
      http:
         url: "https://download.cirros-cloud.net/0.4.0/cirros-0.4.0-x86_64-disk.img"
  pvc:
    accessModes:
      - ReadWriteOnce
    resources:
      requests:
        storage: 100Mi
```
VirtualMachine specification
```
$ cat vm-pvc.yaml
apiVersion: kubevirt.io/v1alpha3
kind: VirtualMachine
metadata:
  labels:
    kubevirt.io/vm: vm-pvc
  name: vm-pvc
spec:
  running: false
  template:
    metadata:
      labels:
        kubevirt.io/vm: vm-pvc
    spec:
      domain:
        devices:
          disks:
          - disk:
              bus: virtio
            name: mydisk
        machine:
          type: ""
        resources:
          requests:
            memory: 1Gi
      terminationGracePeriodSeconds: 0
      volumes:
      - name: mydisk
        persistentVolumeClaim:
          claimName: import-dv-cirros
```

Reply:
@qwang1 - in the YAML example above you specify "" not a volume type other than datavolume. To me, "" says you are taking the default, which seems to be datavolume. I can add that information about default if it's helpful. If you mistakenly use PVC or other type besides datavolume then you'll see a warning. @awels, please correct if I am wrong.

Reply:
@kalexand-rh - Agreed that another place to put this info will offer more exposure. I still think it also belongs here, too as this is a definitive ref list of all the parms. 

Added to Importing-datavolumes module as callout <4> in the YAML:
 Use `type: dataVolume` or `type: ""` (default for dataVolume). If you use other types such as `persistentVolumeClaim`, a warning is triggered and the virtual machine fails to start.